apps/website/src/components/settings/CourseListRow.test.tsx

```diff
@@ -126,7 +152,7 @@ describe('CourseListRow', () => {
   });
 
   it('collapses and expands course details', async () => {
-    const user = userEvent.setup();
+    const user = userEvent.setup({ advanceTimers: vi.advanceTimersByTime });
```

Comment:
the test was failing without this, see https://github.com/testing-library/user-event/issues/1115